.changeset/tough-doors-grow.md

```diff
@@ -0,0 +1,5 @@
+---
+"@primer/css": major
```

Comment:
```suggestion
"@primer/css": minor
```

If we leave the `text-bold` class for now, I think this is a minor change.

Reply:
@jonrohan sounds good, let's try that 👀